### PR TITLE
Convert to ESM

### PR DESCRIPTION
**Pull request by jquense**
I was tired of not being able to ever update of of sindresorhus's packages which all do native esm now

### CODE REVIEW

File: packages/ts-doctor/workspaces.js [hunk 1]
```diff
@@ -1,21 +1,26 @@
...
-const { info, error } = require('@4c/cli-core/ConsoleUtilities');
-const { getPackages } = require('@manypkg/get-packages');
-const commentJson = require('comment-json');
-const get = require('lodash/get');
-const prettier = require('prettier');
-const { parseJsonConfigFileContent, sys } = require('typescript');
+import { error, info } from '@4c/cli-core/ConsoleUtilities';
+import getPkgs from '@manypkg/get-packages';
+import commentJson from 'comment-json';
+import get from 'lodash/get.js';
```
Comment (AaronBuxbaum) [inline packages/ts-doctor/workspaces.js:9]:
```suggestion
import get from 'lodash/get';
```

File: packages/svg-to-component/svgo.config.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-const camelCase = require('lodash/camelCase');
+import camelCase from 'lodash/camelCase.js';
```
Comment (AaronBuxbaum) [inline packages/svg-to-component/svgo.config.js:1]:
```suggestion
import camelCase from 'lodash/camelCase';
```

File: packages/svg-to-component/lib.js [hunk 3]
```diff
@@ -1,15 +1,18 @@
...
-const { chalk } = require('@4c/cli-core/ConsoleUtilities');
-const { transformAsync } = require('@babel/core');
-const yaml = require('js-yaml');
-const camelCase = require('lodash/camelCase');
-const upperFirst = require('lodash/upperFirst');
-const Svgo = require('svgo');
+import { chalk } from '@4c/cli-core/ConsoleUtilities';
+import babel from '@babel/core';
+import { safeLoad } from 'js-yaml';
+import camelCase from 'lodash/camelCase.js';
```
Comment (AaronBuxbaum) [inline packages/svg-to-component/lib.js:9]: is there a good reason for these extensions? seems weird

File: packages/pedantic/lint.js [hunk 4]
```diff
@@ -1,11 +1,11 @@
...
 
-exports.command = '$0 <patterns..>';
+export const command = '$0 <patterns..>';
 
-exports.describe = 'Lint files use ESLint and prettier';
+export const describe = 'Lint files use ESLint and prettier';
 
-exports.builder = (_) =>
-  _.option('fix', {
+export function builder(_) {
```
Comment (AaronBuxbaum) [inline packages/pedantic/lint.js:7]: what's up with this argument being an underscore?

File: packages/cli-core/GitUtilities.js [hunk 5]
```diff
@@ -74,26 +80,32 @@ exports.assertMatchesRemote = async () => {
...
+}
 
-exports.removeTag = (tag) => execa('git', ['tag', '-d', tag]);
-exports.getCurrentBranch = (opts) =>
-  execa('git', ['rev-parse', '--abbrev-ref', 'HEAD'], opts).then(
+export function removeTag(tag) {
+  return execa('git', ['tag', '-d', tag]);
+}
+export function getCurrentBranch(opts) {
+  return execa('git', ['rev-parse', '--abbrev-ref', 'HEAD'], opts).then(
```
Comment (AaronBuxbaum) [inline packages/cli-core/GitUtilities.js:104]: any chance while we're touching this we can move from execa? might be better in a separate PR

**Review by AaronBuxbaum — APPROVED** [review]

File: packages/svg-to-component/lib.js (hunk 3 above)
Comment (jquense) [reply packages/svg-to-component/lib.js:9]: yes they are required for esm imports

File: packages/cli-core/GitUtilities.js (hunk 5 above)
Comment (jquense) [reply packages/cli-core/GitUtilities.js:104]: I don't want to move more things than necessary for this. i'm already pretty unsure i didn't break anything

File: packages/svg-to-component/lib.js (hunk 3 above)
Comment (jquense) [reply packages/svg-to-component/lib.js:9]: well not required for packages if the file is `"main"` or `"exports"` in the package.json sets it up. lodash doesn't set anything up

File: packages/pedantic/lint.js (hunk 4 above)
Comment (jquense) [reply packages/pedantic/lint.js:7]: just means unused
